content/engineering/guest-post/introduction-to-solidjs/index.md

```diff
@@ -0,0 +1,766 @@
+}
+
+p:hover{
+  background: rgb(232, 169, 146);
+  color:#fff;
+  cursor: pointer;
+}
+
+
+.header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+```
+
+And that should give us a much better looking todo app now:
+
+![Updated-app](./updated-app.png)
+
+## SolidJS vs. React
+
+If you're coming from a React background, building the todo app must feel like a piece of cake to you! This is because in terms of syntax and architecture, Solid is strikingly similar to React. It uses the same structure of using functional components, returning JSX, fragments, one-way data binding etc. 
+
+Using effects, refs and events is also much similar to React in Solid. So it's safe to say that Solid provides almost the same if not slightly different syntactic sugar like React. Then why do we need another React like framework for building web applications?
+
+Solid differs from React in terms of performance, developer experience, rendering algorithm, and reactivity. Let's skim through the key differences:
+
+### Performance
+
+In the beginning of this tutorial, we talked how Solid doesn't use a virtual DOM. Instead it uses a compiled DOM. This gives it a significant edge over React as well as literally any other framework in terms of run-time performance. It generates a blazingly fast production bundle that is aimed to provide a more rich user experience to your users. 
+
+![SolidJS Performance](./solidjs-performance.png) 
+
+If you wish to dive deeper into these stats and take a closer look at performance benchmarks, [here's a comparison guide you can refer](https://krausest.github.io/js-framework-benchmark/current.html). 
+
+### Solid is Truly Reactive
+
+Reactivity lies at the heart of SolidJS, which is why you'll never find any unnecessary renders of your Solid components. Only the part of a template that needs to update will re-render.
+
+With React, however, change in a state leads to re-rendering of the entire component. There are ways to combat this, but needs additional efforts in terms of learning these concepts and incorporating them in your applications.
+
+### Community Support, Documentation, and Resources
+
+Even though Solid has a great and extensive documentation, the React community is much larger since it has been around for a longer time. That can be a deal breaker for you if you want to build a scalable web applications quickly. Even third-party libraries are short in number for Solid in comparison to React.
+
+## Conclusion
+
+Solid indeed has an interesting future ahead and is definitely worth a side-project or an MVP for your startup. If you wish to explore Solid in-depth, you can check out an official tutorial by the Solid team here that walks you through each and every concept in Solid. You can also explore the entire code for the demo in this tutorial here. Until next time!
```

Comment:
@FuzzySid Please add the official tutorial link: `official tutorial by the Solid team here`

Reply:
The link has been added now.